src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceStubSettingsClassComposer.java

```diff
@@ -139,7 +139,7 @@
 
   protected static final TypeStore FIXED_TYPESTORE = createStaticTypes();
 
-  private static final VariableExpr DEFAULT_SERVICE_SCOPES_VAR_EXPR =
+  protected static final VariableExpr DEFAULT_SERVICE_SCOPES_VAR_EXPR =
```

Comment:
Could we please move this into another section, separate from the other `private static final`s?

Reply:
moved both `protect static final` to the end of all `private static final`s